lab8-sealed-class.md

```diff
@@ -2,23 +2,36 @@
 
 ## Overview
 
-In this 10-minutes lab, you will get some hands-on experiences with **Sealed Classes** (JEP 397), a preview feature (2nd round in Java 16). This new feature enables to create restricted classes hierarchy. , i.e. restrict which other classes or interfaces may extend or implement them.
+In this 10-minutes lab, you will get some hands-on experiences with **Sealed Classes** (JEP 397), a preview feature (2nd round in Java 16). This new feature enables the ability to create restricted class hierarchies. That is a class or interface can declaratively restrict which other classes or interfaces may extend or implement them.
```

Comment:
That is a class or interface **that**(?) can declaratively...

Reply:
I think both can work, but I'll add "that" to match your stylistic preference.